### PR TITLE
Improve branching documentation for Fedora

### PR DESCRIPTION
**Pull request by jkonecny12**
It's not simple thing to do but hopefully it's understandable this way.

### CODE REVIEW

**Review by poncovka — APPROVED** [review]
Thanks!

File: docs/release.rst [hunk 1]
```diff
@@ -310,6 +342,61 @@ Check if everything is correctly set:
...
+
+::
+
+   git log   # to find SHA of the configuration commit
+   git revert <SHA>  # please add commit message why you are doing the revert
+
+The change revert has to be done if these branch specific configuration changes. Otherwise we won't
+be testing Rawhide on master.
+
+Adjust configuration on master
```
Comment (VladimirSlavik) [inline docs/release.rst:370]: For the other branches you explicitly mention `git checkout ...`, so master should have this too.

File: docs/release.rst [hunk 2]
```diff
@@ -302,6 +298,42 @@ in the Fedora version specific branch on Anaconda side.
...
+
+Adjust shared development branch
+________________________________
+
+
+Switch to f<version>-release branch for Fedora specific settings:
+
+::
+
+   git checkout f<version>-devel
```
Comment (VladimirSlavik) [inline docs/release.rst:327]: (Here too.)

**Review by VladimirSlavik — COMMENTED** [review]
Thank you, this is very useful. However, I think the structure needs some tweaks for consistency.

Apart from these comments, the FIXME seems to be resolved by this change, I think?

**Comment by jkonecny12** [comment]
Hmm, I'm more and more inclined that we should improve this instead of documenting it...

**Comment by jkonecny12** [comment]
Hi @VladimirSlavik, feel free to take this over. Just for explanation, I'm not moving the documentation to the code to reduce the documentation but the main benefit should be that it won't get outdated so easily.

**Pull request closed without merging**